CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+<!--
+******************************************************************************
+* Copyright 2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/-->
+
+# How to Contribute
+
+We welcome community contributions to Intel(R) Extension for Scikit-learn. You can:
+
+- Submit your changes directly with a [pull request](https://github.com/intel/scikit-learn-intelex/pulls).
+- Log a bug or make a feature request with an [issue](https://github.com/intel/scikit-learn-intelex/issues).
+
+Refer to our guidelines on [pull requests](#pull-requests) and [issues](#issues) before you proceed.
+
+## Issues
+
+Use [GitHub issues](https://github.com/intel/scikit-learn-intelex/issues) to:
+- report an issue
+- make a feature request
+
+**Note**: To report a vulnerability, refer to [Intel vulnerability reporting policy](https://www.intel.com/content/www/us/en/security-center/default.html).
+
+## Pull Requests
+
+To contribute your changes directly to the repository, do the following:
+- Make sure you can build the product and run all the examples with your patch.
+- For a larger feature, provide a relevant example.
+- Document your code.
+- [Submit](https://github.com/intel/scikit-learn-intelex/pulls) a pull request into the `master` branch. Provide a brief description of the changes you are contributing.
+
+Public CI is enabled for the repository. Your PR should pass all of our checks. We will review your contribution and, if any additional fixes or modifications are necessary, we may give some feedback to guide you. When accepted, your pull request will be merged into our GitHub repository.
```

Comment:
Should we add something like: "We reserve the right to edit/modify the pull request." Is it in the spirit of an open source product?

Reply:
Do you mean modify the contribution before it is merged or after? If before, then the paragraph above covers it, I think.

If you are talking about future modifications of contributed content, then we can add a note about the license:

![image](https://user-images.githubusercontent.com/17865025/160830766-65b02cd0-aa99-4e73-aec9-57a17dc6d635.png)

Also: I don't think we as a team follow the convention to sign-off our commits but it might be a good idea to do so: 
we can add instructions to add `Signed-off-by: Joe Smith <joe.smith@email.com>` to commits that comply with Developer Certificate of Origin (https://developercertificate.org/)